client/packages/invoices/src/InboundShipment/ListView/LinkInternalOrderModal.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { LinkedRequestRowFragment } from '../api';
+import {
+  useColumns,
+  getNotePopoverColumn,
+  ColumnAlign,
+  DataTable,
+  useWindowDimensions,
+  useTranslation,
+  useDialog,
+  DialogButton,
+} from '@openmsupply-client/common';
+
+interface LinkInternalOrderModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+  requestRequisitions?: LinkedRequestRowFragment[];
+  onRowClick: (row: LinkedRequestRowFragment) => void;
+  isLoading: boolean;
+  onNextClick: () => void;
+}
+
+export const LinkInternalOrderModal = ({
+  isOpen,
+  onClose,
+  requestRequisitions: data,
+  onRowClick,
+  isLoading,
+  onNextClick: createInvoice,
+}: LinkInternalOrderModalProps) => {
+  const t = useTranslation();
+  const { width, height } = useWindowDimensions();
+  const { Modal } = useDialog({ isOpen, onClose });
+
+  const columns = useColumns<LinkedRequestRowFragment>([
+    {
```

Comment:
Have removed the selection box from design and have used onRowClick instead as per our pattern. 

Design also doesn't allow users to create an IS without linking from this modal. Clicking row seems to be

Reply:
Looks great! On first pass, it wasn't obvious to me that the next button would allow you to create a linked in bound shipment. I wonder if maybe some UX to show this better?

Reply:
Nevermind! I just saw now your latest commit adds messaging

Reply:
I am still thinking a different text for next button would be better (and then doesn't require messaging) would be preferred - but this still works great

Reply:
Continue next either way it's not clear and we don't wanna put lots of text into a button